qt/FreeSpaceLabel.cc

```diff
@@ -73,29 +73,37 @@ void FreeSpaceLabel::onTimer()
 
     auto* q = new RpcQueue();
```

Comment:
Since `RpcQueue` is a `QObject`, and `QObject` parents destroy all their children, and all slots/signals are disconnected on `QObject` destruction, would setting a parent here fix the issue?
```suggestion
    auto* q = new RpcQueue(this);
```

Reply:
Looks good to me. That's a much cleaner fix than mine.